sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -70,7 +70,7 @@ class ServiceBusMessage(
     :param body: The data to send in a single message.
     :type body: Optional[Union[str, bytes]]
 
-    :keyword Optional[Dict] application_properties: The user defined properties on the message.
+    :keyword Optional[Dict[str, Any]] application_properties: The user defined properties on the message.
```

Comment:
User property values can by any primitive type? Is there maybe something more specific than `Any` we can use here?

Reply:
I checked in the AmqpAnnotatedMessage here and in EH EventData as well, and it looks like we use `Dict[str, Any]`, `Dict[Union[str, bytes], Any]`, or just `Dict` everywhere.

@yunhaoling - do you know if we can restrict the type here at all?

Reply:
checked the amqp spec: `the values are restricted to be of simple types only, that is (excluding map, list, and array
types)`.